vm-test-build-copy.sh

```diff
@@ -26,7 +26,7 @@ build_and_copy_with_scenarios() {
    sc-meta all build --target-dir $TARGET_DIR --path $contract_path || return 1
    mkdir -p $vm_contract_path/output
    rm -rf $vm_contract_path/scenarios
-   cp $contract_path/output/*.wasm \
```

Comment:
Add a 
`rm $vm_contract_path/output/*.wasm`,
would be nice to automatically get rid of the old files.